content/about/charter.md

```diff
@@ -39,43 +41,49 @@ Responsibilities of each officer of the board are as follows:
 ### Secretary
 
 - Ensure that someone is assigned to and takes minutes during all
-    board EC meetings
-- Revise minutes for publication
+    board meetings
 - Report decisions and action items in formal minutes published
-    publicly (when applicable)
+    publicly
+- Revise minutes for publication, and publish minutes
 
 ### Communications officer
 
 - Receive all communication from community to the board and relay as
   necessary
 
-## Elections
+### Community events officer
+
+- Organise regular community events
 
-Elections will be held at the start of each fall semester. The board
-chair and board secretary are in charge of organizing annual election
-for board renewal.
+### Elections officer
 
-Candidates will self nominate with a short (<300 words) statement that
-will be publicly posted for voters to review.
+- Recruit community members to stand for election
+- Coordinate the election
+- This role should preferably be held by a board member 
+    rotating off the board at the next election
+
+## Elections
 
 The criteria for individuals to stand for election to the board are:
 
 - Is a member of the Carpentry@UiO, broadly defined
 - Has participated in a Carpentries event in any capacity (learner,
-    helper, instructor, organizer, or host) at UiO or elsewhere in the
+    helper, instructor, organizer, board member or host) at UiO or elsewhere in the
     past 2 years
 
+Board members will not serve for more than two consecutive three year terms,
+followed by at least a one year hiatus before serving any new term(s).
+
 Anyone who has participated in a Carpentries event in any capacity
 (learner, helper, instructor, organizer, or host) at UiO or elsewhere in
```

Comment:
This seems somewhat more rigidly defined than the criteria to stand for election to the board above. Intentional? Add "board member" as above?